api/envoy/api/v2/core/base.proto

```diff
@@ -186,6 +186,10 @@ message HeaderValueOption {
   // Should the value be appended? If true (default), the value is appended to
   // existing values.
   google.protobuf.BoolValue append = 2;
+
+  // Whether the header should be modified when it is already present in the request or response.
+  // Defaults to false.
+  google.protobuf.BoolValue skip_if_present = 3;
```

Comment:
Since it defaults false, probably best to just make it a `bool` and skip stating this.